### PR TITLE
Upgrade to v1.16.0 of the KeyCloak Terraform Provider

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@ CHANGELOG
 =========
 
 ## HEAD (Unreleased)
-_(none)_
+* Upgrade to v1.16.0 of the KeyCloak Terraform Provider
 
 ---
+
+## 1.0.0 (2020-02-10)
+* Initial release of the provider against v1.15.0 of the KeyCloak Terraform Provider
```
Comment (mikhailshilkov) [inline CHANGELOG.md]: Nit: I think we should spell it as "Keycloak" everywhere

File: resources.go [hunk 2]
```diff
@@ -36,6 +36,14 @@ const (
 	samlMod   = "Saml"   // the Saml module
```
Comment (mikhailshilkov) [inline resources.go]: Not part of this PR - but aren't these module names supposed to be lowercase? Now, we get Node.js modules called "Saml" etc., which doesn't sound right to me.

Comment (stack72) [reply resources.go]: Ok, let's you and I get together later this week and fix these up and make a new release
